.github/scripts/create-utilix-config.sh

```diff
@@ -5,4 +5,9 @@ cat > $HOME/.xenon_config <<EOF
 rundb_api_url = $RUNDB_API_URL
 rundb_api_user = $RUNDB_API_USER
 rundb_api_password = $RUNDB_API_PASSWORD
+pymongo_url = $PYMONGO_URL
```

Comment:
Perhaps we want to create a new account for this (just to be on the safe side) with only access to the required collection?

Reply:
This is going through the rundb API, not pymongo. I needed to add the pymongo variables (which are readonly in the secrets) to set up the contexts.

Reply:
I think that wasn't my question, it was related to if we can somehow add a new user that is only used here, in the git actions.

Not saying we should, just a question/suggestion.

Reply:
Ah sorry I guess I misunderstood; yes we could absolutely make another user. But I wanted to clarify that the pymongo strings are here to be able to instantiate the different contexts, so this new user would need to have permissions to access all the needed collections (I guess there are two? Not sure exactly), not the context collection.